dataprofiler/profilers/float_column_profile.py

```diff
@@ -71,6 +124,12 @@ def profile(self):
         if self.histogram_selection is not None:
             histogram_method = self.histogram_selection
 
+        protected_precision = {}
+        for key in ['min', 'max']:
+            protected_precision[key] = None
+            if self.precision[key] is not None:
+                protected_precision[key] = int(self.precision[key])
```

Comment:
So what exactly is the point of this? If the max and min are calculated by taking the min and max over a list of string lengths, isn't it already guaranteed that they'll be integers?

Reply:
They're all of float type